src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -37,6 +40,8 @@
 
     public static final int DEFAULT_PLOIDY = HomoSapiensConstants.DEFAULT_PLOIDY;
 
+    private static final GenotypeLikelihoodCalculators GL_CALCS = new GenotypeLikelihoodCalculators();
```

Comment:
Yes, there is a shared cache but this cache is filled completely in the constructor of `GL_CALCS` and afterwards never mutates.  Thus there is no danger in sharing it between threads.